style.css

```diff
@@ -781,3 +802,122 @@ button:hover {
     width: 50%;
   }
 }
+
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  border: 1px solid black;
+  border-radius: 10px;
+  z-index: 10;
+  background-color: white;
+  width: 600px;
+  max-width: 80%;
+}
```

Comment:
- [ ] Your popup window design I can't see the project title title and close button and  showing like that in desktop version.

### Your Design

![image](https://github.com/OrdinaryMack/Portfolio-Contact-form/assets/118582316/6fb3725c-d998-4b2c-904f-9675d66aeaf0)

### Figma Design

![Screenshot (203)](https://github.com/OrdinaryMack/Portfolio-Contact-form/assets/118582316/a41da7ff-6d9d-4c98-87bc-39d5f3038e73)